packages/design-system/src/new-ui/Icons/Dot.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from "react";
+import { IconBase, IconBaseProps } from "./IconBase";
+
+export const Dot = React.forwardRef<
+  SVGSVGElement,
+  Omit<IconBaseProps, "viewBox" | "children">
+>((props, ref) => {
+  const { className, ...passThrough } = props;
+  return (
+    <IconBase
+      {...passThrough}
+      ref={ref}
+      viewBox="0 0 24 24"
+      className={className}
+    >
+      <circle cx="5" cy="5" r="4" fill="white" />
```

Comment:
@iamnamananand996 

Fill is not working in this icons set, all the icons are doing thing with path, please move this icons to ComplicatedIcons for a temp solution

Reply:
@iamnamananand996 Please delete this dot icon and we are good to go